umh.docs.umh.app/content/en/docs/features/datainfrastructure/postgresql_and_grafana_alerts.md

```diff
@@ -35,13 +35,13 @@ concentration of a component is too low. Notifications can be sent
 simultaneously across many services like Discord, Mail, Slack, Webhook,
 Telegram, or [Microsoft Teams](https://learn.umh.app/blog/how-to-install-grafana-alerts-in-microsoft-teams/). It is also possible to forward the alert with
 SMS over a personal Webhook. A complete list can be found on the
-[Grafana page](https://grafana.com/docs/grafana/latest/alerting/manage-notifications/manage-contact-points/configure-integrations/)
+[Grafana page](https://grafana.com/docs/grafana/latest/alerting/)
 about alerting.
 
 ## How can I use it?
```

Comment:
Can we integrate the learn article in here, and use our new data source in the new data model for it?